docs/docs/03-tools/05-context.md

```diff
@@ -0,0 +1,64 @@
+# Context
+
+GPTScript supports context provider tools which can be used to provide additional context for tool execution. Output from the context tool gets prepended to the instruction of the current tool that uses the context tool.
```

Comment:
We shouldn't use "context provider tools" in the definition as those are not yet defined.

I think we are trying to convey that context:
- Provides a mechanism to use prompt information across many tools as an import mechanism.
- Provide additional information to the calling tool on when to use a specific tool. (Helpful for using new CLIs for example)
- These can be static txt or dynamic gptscripts.

Reply:
Following documentation has been added now:'
```
GPTScript provides a mechanism to share prompt information across many tools using the tool parameter `context`. It is used to provide additional information to the calling tool on when to use a specific tool by prepending the `context` to the instruction of the calling tool.

- Context can point to a static text or a GPTScript.
- Context tools are just regular GPTScript tools, and any valid gptscript field can be used.
- Exported tools from a context tools are available as tools to the calling tool.
- When context points to a GPTScript tool, output from the context tool gets prepended to the instruction of the calling tool.
```